scripts/generate-cli-readme

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env node
+
+// WARNING:
+// 
+// This generator is very opinated about what is and isn't the right way to have a 
+// CLI help section. If you want to know the preferred way to write CLI help sections, 
+// look at `cfs-cli` or `ara-network-node-dcdn`
+
+const { execFile } = require('child_process')
+const { resolve } = require('path')
+const pify = require('pify')
+const fs = require('fs')
+
+let commandSections = []
+
+const PROJECT_NAME = process.argv[2]
```

Comment:
throw error / show help if no argument

Reply:
playing with this a little more, maybe this should just be plucked from the dir structure?  if the name differs from what the bin file is called, it doesn't work (at least not on macos).